helm/squidex7/values.yaml

```diff
@@ -224,6 +233,12 @@ mongodb-replicaset:
   ##
   replicas: 3
   
+  ## @skip podSecurityContext
+  podSecurityContext: { }
```

Comment:
Should this not be described?

Reply:
Yes, I will add annotations to the yaml soon.

Reply:
Described.